bundles/binaries-parent/build.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project name="org.eclipse.swt.cocoa.macosx.x86_64" default="build.jars" basedir=".">
+<project name="org.eclipse.swt.${ws}.${os}.${arch}" default="build.jars" basedir=".">
```

Comment:
I wonder if the name has to match the platform or if this is irrelevant?

Reply:
Ant's project name shouldn't matter but let's have it different for the case when we have to debug smth.